omaha/base/constants.h

```diff
@@ -153,13 +153,13 @@ const TCHAR* const kTemporaryFilenamePrefix = _T("gup");
 // Omaha's app ID
 //
 // TODO(omaha): Rename all of these "Omaha".
-#define GOOPDATE_APP_ID _T("{430FD4D0-B729-4F61-AA34-91526481799D}")
+#define GOOPDATE_APP_ID _T("{1E972C4E-6353-4467-BF19-94DA1127DEAC}")
```

Comment:
this app ID will change base the app as far as I understand

Reply:
This is guid of omaha update itslef. It will be used if we need to update our update system :)